frontend/components/TableContainer/DataTable/TooltipTruncatedTextCell/TooltipTruncatedTextCell.tsx

```diff
@@ -56,7 +60,7 @@ const TooltipTruncatedTextCell = ({
         delayHide={200} // need delay set to hover using clickable
       >
         <>
-          {value}
+          {tooltip !== undefined ? tooltip : value}
```

Comment:
Could this be written as `{tooltip || value}`

Reply:
nice! @ghernandez345 suggested we go with `??`. Fixed now